/var/spool/slurmd/job55340/slurm_script: line 11: cd: SSL_VAN: No such file or directory

----------------------------------------------------------------------------
  cuda:
----------------------------------------------------------------------------
     Versions:
        cuda/10.2
        cuda/11.0
        cuda/11.2
        cuda/11.3
        cuda/11.4
        cuda/11.5
        cuda/11.6
        cuda/12.0

----------------------------------------------------------------------------
  For detailed information about a specific "cuda" package (including how to load the modules) use the module's full name.
  Note that names that have a trailing (E) are extensions provided by other modules.
  For example:

     $ module spider cuda/12.0
----------------------------------------------------------------------------

 


CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:112: UserWarning: Found total gpus 2
  warnings.warn("Found total gpus {}".format(args.ngpus_per_node))
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/dictionary.py:196: UserWarning: `threshold_values=True/False` is deprecated, please use `threshold=value` instead.
  warnings.warn("`threshold_values=True/False` is deprecated, please use `threshold=value` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:219: UserWarning: 0 gpu 0
  warnings.warn(f"{args.rank} gpu {args.gpu}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:221: UserWarning: Batch size is: 4 epochs 15000 and task is Task03_Liver
  warnings.warn(f"Batch size is: {args.batch_size} epochs {args.max_epochs} and task is {args.task}")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/dictionary.py:196: UserWarning: `threshold_values=True/False` is deprecated, please use `threshold=value` instead.
  warnings.warn("`threshold_values=True/False` is deprecated, please use `threshold=value` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:219: UserWarning: 1 gpu 1
  warnings.warn(f"{args.rank} gpu {args.gpu}")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/array.py:176: UserWarning: `to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.
  warnings.warn("`to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:287: UserWarning: Total parameters count 33248936
  warnings.warn(f"Total parameters count {pytorch_total_params}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:290: UserWarning: Total args.checkpoint True
  warnings.warn(f"Total args.checkpoint {args.checkpoint}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:298: UserWarning:  Best url model is 64-128-256-512_3-4-6-3_8-8-4-4_vae___VANV4_Task03_Liver__best.pt, final model url is 64-128-256-512_3-4-6-3_8-8-4-4_vae___VANV4_Task03_Liver__final.pt
  warnings.warn(f" Best url model is {args.best_model_url}, final model url is {args.final_model_url}")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/array.py:176: UserWarning: `to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.
  warnings.warn("`to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:287: UserWarning: Total parameters count 33248936
  warnings.warn(f"Total parameters count {pytorch_total_params}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:290: UserWarning: Total args.checkpoint True
  warnings.warn(f"Total args.checkpoint {args.checkpoint}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:298: UserWarning:  Best url model is 64-128-256-512_3-4-6-3_8-8-4-4_vae___VANV4_Task03_Liver__best.pt, final model url is 64-128-256-512_3-4-6-3_8-8-4-4_vae___VANV4_Task03_Liver__final.pt
  warnings.warn(f" Best url model is {args.best_model_url}, final model url is {args.final_model_url}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:137: UserWarning: Writing Tensorboard logs to ./runs/MSD/test_log
  warnings.warn(f"Writing Tensorboard logs to {args.logdir}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:145: UserWarning: 0  Sat May 20 11:37:24 2023  Epoch: 0
  warnings.warn(f"{args.rank}  {time.ctime()}  Epoch: {epoch}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:145: UserWarning: 1  Sat May 20 11:37:24 2023  Epoch: 0
  warnings.warn(f"{args.rank}  {time.ctime()}  Epoch: {epoch}")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 93), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 86), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 94), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 198), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 239), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 247), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 333), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 237), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 427), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 455), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 685), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 771), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 761), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/spatial/array.py:550: UserWarning: axcodes ('RAS') length is smaller than the number of input spatial dimensions D=2.
Orientation: input spatial shape is (512, 836), num. channels is 512,please make sure the input is in the channel-first format.
  warnings.warn(
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/MSD/main.py", line 370, in <module>
    main()
  File "/home/karimimonsefi.1/SSL_VAN/MSD/main.py", line 114, in main
    mp.spawn(main_worker, nprocs=args.ngpus_per_node, args=(args,))
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 240, in spawn
    return start_processes(fn, args, nprocs, join, daemon, start_method='spawn')
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 198, in start_processes
    while not context.join():
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 160, in join
    raise ProcessRaisedException(msg, error_index, failed_process.pid)
torch.multiprocessing.spawn.ProcessRaisedException: 

-- Process 1 terminated with the following error:
Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 69, in _wrap
    fn(i, *args)
  File "/home/karimimonsefi.1/SSL_VAN/MSD/main.py", line 348, in main_worker
    accuracy = run_training(
  File "/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py", line 147, in run_training
    train_loss = train_epoch(
  File "/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py", line 23, in train_epoch
    for idx, batch_data in enumerate(loader):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 628, in __next__
    data = self._next_data()
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1333, in _next_data
    return self._process_data(data)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1359, in _process_data
    data.reraise()
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/_utils.py", line 543, in reraise
    raise exception
RuntimeError: Caught RuntimeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 89, in apply_transform
    return _apply_transform(transform, data, unpack_items)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 53, in _apply_transform
    return transform(parameters)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/croppad/dictionary.py", line 893, in __call__
    d[key] = self.cropper.crop_pad(img=d[key], box_start=box_start, box_end=box_end, mode=m)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/croppad/array.py", line 763, in crop_pad
    pad_to_end = np.maximum(box_end - np.asarray(img.shape[1:]), 0)
ValueError: operands could not be broadcast together with shapes (2,) (3,) 

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 89, in apply_transform
    return _apply_transform(transform, data, unpack_items)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 53, in _apply_transform
    return transform(parameters)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/compose.py", line 173, in __call__
    input_ = apply_transform(_transform, input_, self.map_items, self.unpack_items, self.log_stats)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 113, in apply_transform
    raise RuntimeError(f"applying transform {transform}") from e
RuntimeError: applying transform <monai.transforms.croppad.dictionary.CropForegroundd object at 0x1490cead1e80>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/_utils/worker.py", line 302, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 58, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 58, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/data/dataset.py", line 97, in __getitem__
    return self._transform(index)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/data/dataset.py", line 83, in _transform
    return apply_transform(self.transform, data_i) if self.transform is not None else data_i
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 113, in apply_transform
    raise RuntimeError(f"applying transform {transform}") from e
RuntimeError: applying transform <monai.transforms.compose.Compose object at 0x1490ceaea1f0>


